File: archive/find-first-palindromic-string-in-the-array.js
const firstPalindrome = function(words) {
  let ans = ''
  for (let word of words) {
    if (word === word.split('').reverse().join('')) {
      ans = word
      break
    }
  }

  return ans
};

console.log(firstPalindrome(["notapalindrome","racecar"]))

File: go-archive/find-first-palindromic-string-in-the-array.go
func firstPalindrome(words []string) string {
    for _, v:= range words {
        if (isPalindromic(v)) {return v}
    }

    return ""
}

func isPalindromic (word string) bool {
    for i:=0; i < len(word); i++ {
        if (word[i] != word[len(word) - 1 - i]) {return false}
    }

    return true
}